Production/crab/config/2017/non_resonant.txt

```diff
@@ -0,0 +1,4 @@
+Automatic
```

Comment:
- Automatic shouldn't be specified
- why this config is here? (there is another config with all non-resonant samples)